frontend/src/components/seller/BidCreateModal.tsx

```
@@ -0,0 +1,253 @@
+        setIsLoading(true);
+        setQuote(null);
+        const quoteData = await getQuoteDetail(quoteId);
+        if (ignore || requestId !== loadQuoteRequestIdRef.current) return;
+        setQuote(quoteData);
+      } catch (error) {
+        if (ignore || requestId !== loadQuoteRequestIdRef.current) return;
+        logError("견적 조회 실패:", error);
+        toast.error("견적 정보를 불러오는데 실패했습니다.");
+        onClose();
+      } finally {
+        if (ignore || requestId !== loadQuoteRequestIdRef.current) return;
```

Biome (2.4.9)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)